src/languages/powershell.js

```diff
@@ -42,6 +42,23 @@ export default function(hljs){
     // TODO: 'validate[A-Z]+' can't work in keywords
```

Comment:
IE, you'd add a `built_in` key here...

Reply:
So, something like this?
```js
  var KEYWORDS = {
   keyword: '...',
   built_in: 'ac asnp cat cd CFS chdir clc...'
  };

Reply:
Yes.

https://highlightjs.readthedocs.io/en/latest/reference.html#keywords

Reply:
I'm not exactly sure what PS_METHODS is trying to do.

Reply:
> Yes.
> 
> https://highlightjs.readthedocs.io/en/latest/reference.html#keywords

I see! I'll make the appropriate changes.